shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/ApiMapper.java

```diff
@@ -72,4 +79,44 @@ public interface ApiMapper {
      * @return update count
      */
     int updateByPrimaryKey(ApiDO record);
+
+    /**
+     * existed.
+     *
+     * @param id id
+     * @return existed
+     */
+    @Override
+    Boolean existed(@Param("id") Serializable id);
+
+    /**
+     * select api by query.
+     *
+     * @param query {@linkplain ApiQuery}
+     * @return {@linkplain List}
+     */
+    List<ApiVO> selectByQuery(ApiQuery query);
```

Comment:
change to `List<ApiDO> selectByQuery(ApiQuery query);`

Reply:
fixed it